src/drivers/imx/edma.c

```diff
@@ -345,6 +345,7 @@ static int edma_set_config(struct dma_chan_data *channel,
 	trace_edma("EDMA: set config");
 
 	channel->is_scheduling_source = config->is_scheduling_source;
+	channel->direction = config->direction;
 
```

Comment:
@paulstelian97 this doesn't seem related to this change. So, please put in another patch.